nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/varian/VarianData.java

```diff
@@ -762,6 +766,30 @@ public boolean isComplex(int iDim) {
         }
     }
 
+    @Override
+    public int getGroupSize(int iDim) {
+        if (iDim == 0) {
+            String s = getPar("proc");
+            return !(s != null && s.equals("rft")) ? 2 : 1; // proc="ft" or "lp"
```

Comment:
I think this is a bit complicated to read with all the negatives, just double check that I have adjusted it correctly so that if s is rft then the group size is 1, otherwise it is 2. 
```suggestion
            return "rft".equals(s) ? 1 : 2; // proc="ft" or "lp"
```